cmd/codegen/generator/go/generator.go

```diff
@@ -319,7 +361,7 @@ func generateCode(
 	fset *token.FileSet,
 	pass int,
 ) error {
-	funcs := templates.GoTemplateFuncs(ctx, schema, schemaVersion, cfg.ModuleName, cfg.ModuleParentPath, pkg, fset, pass)
+	funcs := templates.GoTemplateFuncs(ctx, schema, schemaVersion, cfg.ModuleName, cfg.ModuleParentPath, pkg, fset, pass, cfg.ClientOnly, cfg.LocalSDK)
```

Comment:
:thinking: this is getting a bit long now, probably worth just passing in `cfg` directly.